### PR TITLE
included all the requirements for the build of images

### PR DESCRIPTION
**Pull request by harshad16**
included all the requirements for the build of images
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Description

These files will help with the s2i build.
we can have GPU based image build via aicoe-ci due to licensing.

### CODE REVIEW

File: overlays/ps-nlp-tensorflow-gpu/Pipfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ nltk = "*"
 gensim = "*"
 spacy = "*"
 keras = "*"
+jupyter-tensorboard = "*"
```
Comment (pacospace) [inline overlays/ps-nlp-tensorflow-gpu/Pipfile:11]: why does this need to be a direct dependency?

**Comment by pacospace** [comment]
Related-to: https://github.com/thoth-station/ps-nlp/issues/3

File: overlays/ps-nlp-tensorflow-gpu/Pipfile (hunk 1 above)
Comment (pacospace) [reply overlays/ps-nlp-tensorflow-gpu/Pipfile:11]: isn't it a transitive dependency? It should be part of an advise this information?

Comment (harshad16) [reply overlays/ps-nlp-tensorflow-gpu/Pipfile:11]: sorry, I m not aware if it is a transitive dependency of any package, can you please share the package :grinning: 

>It should be part of an advise this information?
you mean by thoth advise.

Comment (pacospace) [reply overlays/ps-nlp-tensorflow-gpu/Pipfile:11]: I see you are adding `jupyter-tensorboard`, is there an error in the build process if this is missing? or is there any other reason for having that? If that is the case, should that information be part of a reccomendation, in other words, should we have a new justification?

Comment (harshad16) [reply overlays/ps-nlp-tensorflow-gpu/Pipfile:11]: this is not based on error.
this is a plugin for tensorboard to show up in the jupyter.

**Comment by pacospace** [comment]
@harshad16 based on inputs from https://github.com/thoth-station/ps-nlp/issues/3, we need also transformers package from hugging face: https://huggingface.co/transformers/installation.html

**Comment by harshad16** [comment]
>  based on inputs from #3, we need also transformers package from hugging face: https://huggingface.co/transformers/installation.html

@pacospace added these packages

> There is also a requirement for some small models for NLP: [#3 (comment)](https://github.com/thoth-station/ps-nlp/issues/3#issuecomment-866779242)

I think, we should check on next pr, as i feel ntlk package could handle what is require.
if not then we can add more in next pr.

File: overlays/ps-nlp-pytorch/Pipfile [hunk 2]
```diff
@@ -7,8 +7,12 @@ name = "pypi"
 nltk = "*"
 gensim = "*"
 spacy = "*"
+jupyter-tensorboard = "*"
+tensorboard = "<2.0"
 torch  = "*"
 torchvision= "*"
+huggingface = "*"
```
[the remark this answers is not in the document]
Comment (harshad16) [reply overlays/ps-nlp-pytorch/Pipfile]: ack, this github repo seems to be deleted.
i have removed the package from all.

**Review by pacospace — APPROVED** [review]
Thanks!!! 💯

**Pull request merged**